firmware/ventilator-controller-stm32/Core/Src/Pufferfish/Driver/BreathingCircuit/ControlLoop.cpp

```diff
@@ -43,6 +47,18 @@ void HFNCControlLoop::update(uint32_t current_time) {
   // TODO(lietk12): we should probably set flow to NaN otherwise, but for now we do nothing
   // so that we don't overwrite the simulated values if the sensors aren't available
 
+  if (air_status == InitializableState::ok) {
+    sensor_connections_.sfm3019_air_connected = true;
+  } else if (air_status == InitializableState::failed) {
+    sensor_connections_.sfm3019_air_connected = false;
+  }
```

Comment:
Are there any other possible `InitializableState` values to handle? If not, could we just do `sensor_connection_.sfm3019_air_connected = air_status == InitializableState::ok`?

Reply:
> Are there any other possible `InitializableState` values to account for? 

Nope!
If not, could we just do `sensor_connection_.sfm3019_air_connected = air_status == InitializableState::ok`?

yep done.